CONTRIBUTING.md

```diff
@@ -1,59 +1,91 @@
+2. Always take a pull from the remote repository to your master branch to keep it at par with the main project(updated repository).
+        
+        git pull upstream master
+        
+3. Create a branch. For example, if you are going to work on issue number #44 (which is, say, a new feature for ‘forgot password’ management):
 
         git checkout -b forgot-password#44
 
     This both creates and checks out that branch in one command.  
     The feature name should provide a (short) description of the issue.
 
-2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
-   
-3. Develop the feature and push it to your local repo.
-   
-4. Create Pull Request (PR). Make sure to comment the issue that your PR is supposed to solve.
+4. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+5. Commit your changes and push it to your fork of the repository.
+6. Create Pull Request (PR). Make sure to comment the issue that your PR is supposed to solve.
 
-## Community
-Discussions about CircuitVerse issues and features take place on the repository's [Issues](https://github.com/CircuitVerse/CircuitVerse/issues) and [Pull Requests](https://github.com/CircuitVerse/CircuitVerse/pulls) sections. Anybody is welcome to join these conversations. See the [README](README.md) for more information on communication channels.
+## Create a pull request
+
+- Try to keep the pull requests small. A pull request should try its very best to address only a single concern.
+- For work in progress pull requests, please use the Draft PR feature.
+- Make sure all tests pass and add additional tests for the code you submit.
+- Document your reasoning behind the changes. Explain why you wrote the code in the way you did. The code should explain what it does.
+- If there's an existing issue, reference to it by adding something like `References/Closes/Fixes/Resolves #123`, where 123 is the issue number. 
+- Please fill out the PR Template when making a PR.
+
+> Please note: a core team member may close your PR if it has gone stale or if we don't plan to merge the code.
 
-## Contributors
-### Financial Contributors
+## Pull request reviews
 
-Becoming a [financial contributor](https://opencollective.com/CircuitVerse/contribute) helps us sustain our community and platform.
+- Requested Changes must be resolved (with code or discussion) before merging.
+- If you make changes to a PR, be sure to re-request a review.
+- Don't repeadetely tag someone(may be it is not the right time to review your PR), be patient.
+- Do not 'resolve conversation' unnecessary raised by communtiy member or any workflows tools(codeclimate or hound) as they may have some purpose, try to resolve the request changes and if any help wanted tag communtiy member to give views about that.
```

Comment:
Hey @vedant-jain03, I think by mistake you have written the 'communtiy' in place of 'community' ...